docs/reference/search-api.md

```diff
@@ -53,13 +53,14 @@ Search for documents matching a query in the given index `<index name>`.
 
 #### Get parameters
 
-| Variable                  | Type                 | Description                                                                                       | Default value                                                                                   |
-| ------------------------- | -------------------- | ------------------------------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------------- |
-| **query**                     | `String`               | Query text. See the [query language doc](query-language.md) (mandatory)                                          |                                                                                                |
-| **startTimestamp**                 | `i64`              | If set, restrict search to documents with a `timestamp >= start_timestamp`                                                            |                                                                                |
-| **endTimestamp**                 | `i64`              | If set, restrict search to documents with a `timestamp < end_timestamp``                                                            |                                                                                     |
-| **startOffset**                | `Integer`              | Number of documents to skip                                                                | `0`                                                                                             |
-| **maxHits**                 | `Integer`              | Maximum number of hits to return (by default 20)                                                            | `20`                                                                                            |
+| Variable | Type | Description | Default value |
```

Comment:
why is there a diff here?

Reply:
Fine, generally it's better to isolate the formatting from other changes because diff are useless then. But that's ok for this one ;)

Reply:
Ok, for cross check these are the lines I added:
```
| **searchFields** | `[String]` | If set, specify the set of fields the search will be performed on | |
| **format** | `String` | Response output format. `json` or `pretyjson`  | `pretyjson` |
```